deploy-manage/deploy/self-managed/vm-max-map-count.md

```diff
@@ -12,10 +12,14 @@ products:
 
 {{es}} uses a [`mmapfs`](elasticsearch://reference/elasticsearch/index-settings/store.md#mmapfs) directory by default to store its indices. The default operating system limits on mmap counts is likely to be too low, which may result in out of memory exceptions.
```

Comment:
We say here "The default operating system limits on mmap counts is likely to be too low" but that is no longer true. I think your addition below supersedes this statement and we can remove it.

Reply:
Maybe we want to say "could be too low"?